grants_tagger_light/training/train.py

```diff
@@ -129,27 +112,66 @@ def sklearn_metrics(prediction: EvalPrediction):
 
     metrics = trainer.evaluate(eval_dataset=val_dset)
 
-    print(metrics)
+    logger.info(pformat(metrics))
```

Comment:
do we write the metrics anywhere other than the logs? if not lets do

Reply:
one option is to include them in the config, maybe there are there already?

Reply:
They are logged to wandb

Reply:
we need them somewhere in the repo, we dont use wandb to find results. so they either need to be recorded in dvc or in the config.